functions/defaulted_parameters.md

```diff
@@ -0,0 +1,87 @@
+# Module name: topic name
+_Skeleton instructions are typeset in italic text._
+
+## Overview
+
+Functions in C++ may be overloaded with different numbers and types of parameters. It may be of value to specify default values for some number of parameters, to allow a consumer to avoid specifying parameters that rarely change, or to enable expanding the set of parameters while maintaining backward compatibility with existing consumers.
+
+<table>
+  <thead>
+    <th>Level</th>
+    <th>Objectives</th>
+  </thead>
+  <tr>
+    <td>Foundational knowledge</td>
+    <td>can make calls to functions with or without defaulted parameters</td>
+  </tr>
+  <tr>
+    <td>Advanced</td>
+    <td>declares and defines functions with defaulted parameters</td>
+  </tr>
+  <tr>
+    <td>Further studies</td>
+    <td>refinement of defaulted parameters through multiple declarations</td>
+  </tr>
+</table>
+
+## Motivation
+
+Default parameters allow the omission of parameters with obvious or common values. Also may be utilized to extend an existing function signature without forcing changes to existing calling code.
+
+## Topic introduction
+
+Explain how default parameters work and how to define them.
+
+## Foundational knowledge: Using *
+
+### Background/Required Knowledge
+
+A student is able to:
+
+* Make calls to existing functions, passing parameters
```

Comment:
```suggestion
* Make calls to existing functions, passing parameters  [[Functions: calling functions]][1]
```
Where ever we can we should add references to other topics with links, so it is easier to navigate the different topics afterwards.
`[1]` refers to the link I suggested to add at the bottom of the file

Reply:
done